go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/prometheus/tsdb v0.7.1
 	github.com/rs/zerolog v1.22.0
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.7.0 // indirect
```

Comment:
Should not be needed in the `go.mod` file.

Reply:
It seemed weird to me that the dependency wasn't in go mod already, but without it I get an error:

```
go: github.com/optakt/flow-dps/tests/integration/rosetta:  package github.com/stretchr/testify/assert imported from  implicitly required module; to add missing requirements, run:
        go get github.com/stretchr/testify/assert@v1.7.0
go: github.com/optakt/flow-dps/tests/integration/rosetta:  package github.com/stretchr/testify/require imported from  implicitly required module; to add missing requirements, run:
        go get github.com/stretchr/testify/require@v1.7.0
```
How come it isn't needed then?

Reply:
Nevermind that :p It shouldn't be indirect though, but this will be solved once you rebase on master since on master we directly depend on it.